server/lib/recurring-contributions.js

```diff
@@ -31,13 +31,22 @@ export const MAX_RETRIES = 6;
 export async function ordersWithPendingCharges({ limit, startDate } = {}) {
   return models.Order.findAndCountAll({
     where: {
+      status: { [Op.not]: status.PAUSED },
```

Comment:
Surprise that we don't have already a condition there. Because we rely on Subscription.isActive? Should we tweak that one too?

Reply:
Below, in the `include`, we have another condition that filters out inactive/external subscriptions: https://github.com/opencollective/opencollective-api/blob/fa67633a8a0789e39a418ae2106507ce26465d7d/server/lib/recurring-contributions.js#L56